server/controllers/authController.ts

```diff
@@ -6,9 +6,10 @@ import { CookieOptions } from '../../shared/types/CookieOptions';
 const cookieOptions: CookieOptions = {
   expires: new Date(
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    Date() + +process.env.JWT_COOKIE_EXPIRES_IN! * 24 * 60 * 60 * 1000,
+    Date.now() + +process.env.JWT_COOKIE_EXPIRES_IN! * 24 * 60 * 60 * 1000,
```

Comment:
Add this variable to `.env.example` 
if it's number of hours add this to name. e.g. `JWT_COOKIE_EXPIRES_IN_HOURS`

Reply:
Fixed